.nycrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We specify these numbers in the `nyc` property in `package.json` already so we don't need to add a new file for this.